chaosazure/vmss/actions.py

```diff
@@ -248,34 +248,47 @@ def choose_vmss_instance_at_random(vmss_choice, configuration, secrets):
         logger.debug(
             "Found virtual machine scale set instances: {}".format(
                 [x['name'] for x in vmss_instances]))
-    choice_vmss_instance = random.choice(vmss_instance)
+    choice_vmss_instance = random.choice(vmss_instances)
     return choice_vmss_instance
 
-def choose_vmss_instance(vmss_choice, configuration, secrets, instance_criteria):
+def choose_vmss_instance(vmss_choice: dict, 
+                         configuration: Configuration = None,
+                         instance_criteria: Iterable[Mapping[str, any]] = None,
+                         secrets: Secrets = None):
```

Comment:
Missing a typing annotation of the return type. Maybe something like `Dict[str, Any]` ?